make_release/Readme.md

```diff
@@ -1,8 +1,22 @@
 # The release process of Nushell
 ## 0. Check dependencies
-- [ ] [`reedline`] is updated and released ([example][reedline bump example])
-- [ ] pin [`reedline`] in [`nushell`] ([example][reedline pin example])
-- [ ] [`nu-ansi-term`] stays the same
+> **Note**  
+> the following procedure is the same for `nu-ansi-term` and `reedline` and needs to be repeated
+
+> **Warning**  
+> release `nu-ansi-term` **before** `reedline` and `reedline` **before** Nushell
+
+> **Note**  
+> `nu-ansi-term` is typically released only when there are changes to publish.
+> `reedline` is typically released on the same schedule as Nushell.
```

Comment:
I think we had the wrong strategy atleast for `nu-ansi-term`.
I think we were somewhat overzealous with publishing nu-ansi-term changes and the crate should be and behave as stable as possible as we interact with external dependencies that also pull in `nu-ansi-term` like `lscolors`. Just pushing out `nu-ansi-term` the day before would fail with that. (`nu-ansi-term` should be published and accepted by the relevant deps beforehand. Only then we can version bump in nu. And to get good will from external users we should be deliberate about releases.)

We should follow for both a bit more of a pull model if we don't actively know we need to publish them.

So the thing worth checking is ther a git dep and then considering the available commits. (if there hasn't been a git dep the changes haven't been pulled in and tested, which would most likely be to late the day before)

Reply:
so should i just merge these two sentences into something like 
```md
> both `reedline` and `nu-ansi-term` is typically released only when there are changes to publish.
```
?

Reply:
Mhh I think it is worth keeping in mind that we need to be in sync with the ecosystem around `nu-ansi-term`. That was a goof a release ago.

Reply:
okey, we can rework that part a bit later, we have some time to refine the document :relieved: